src/bin/fullnode.rs

```diff
@@ -81,9 +87,18 @@ fn main() {
     // entry_height is the network-wide agreed height of the ledger.
     //  initialize it from the input ledger
     eprintln!("processing ledger...");
-    let entry_height = bank.process_ledger(entries).expect("process_ledger");
+    let entry_height = bank.process_ledger(entries.clone())
+        .expect("process_ledger");
     eprintln!("processed {} ledger...", entry_height);
 
+    let window_entries = {
```

Comment:
How about a `split_at()` before `process_ledger()` so that you don't need to `clone()` it?

Reply:
please ignore the .collect()...  deemed infeasible in Discord#development discussion.  therefore, split_at() non-option